WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -157,6 +158,24 @@ def wca_table_for(records, hover: true, striped: true, extra_table_class: "", &b
           "Organizer(s)"
         end
 
+        table.define :date do |competition|
+          date_range(competition.start_date, competition.end_date, separator: '-', show_year: false)
+        end
+        table.define :date_header do
+          "Date"
+        end
+
+        table.define :results_header do
+          ""
+        end
+
+        table.define :country_city do |competition|
+          wca_highlight competition.country_name + ", " + competition.cityName, competition.country_name
+        end
+        table.define :country_city_header do
+          "Country, City"
+        end
+
```

Comment:
Just curious, since you're not using wca_table_for anymore, are these actually being used?